backend/dataall/modules/worksheets/services/worksheet_service.py

```diff
@@ -135,7 +135,7 @@ def delete_worksheet(uri) -> bool:
     def run_sql_query(uri, worksheetUri, sqlQuery):
         with get_context().db_engine.scoped_session() as session:
             environment = EnvironmentService.get_environment_by_uri(session, uri)
-            worksheet = WorksheetService._get_worksheet_by_uri(session, worksheetUri)
+            worksheet = WorksheetService.get_worksheet(uri=worksheetUri)
```

Comment:
any reason this and not `@ResourcePolicyService.has_resource_permission(GET_WORKSHEET, param_name='worksheetUri`)`

Reply:
I was not aware of that functionality! I just changed it now